src/Components/Facility/BedManagement.tsx

```diff
@@ -176,22 +207,38 @@ const BedRow = (props: BedRowProps) => {
             </Button>
           </div>
         ) : (
-          <Button
-            color="inherit"
-            variant="contained"
-            type="submit"
-            size="small"
-            style={{
-              marginLeft: "auto",
-              backgroundColor: "#24a0ed",
-              color: "white",
-            }}
-            onClick={() => setIsEditable(true)}
-          >
-            EDIT
-          </Button>
+          <div>
+            <Button
+              color="inherit"
+              variant="contained"
+              type="submit"
+              size="small"
+              className="fml-auto bg-[#24a0ed] text-white mx-2"
+              onClick={() => setIsEditable(true)}
+            >
+              <i className="fas fa-pencil-alt text-white mr-2"></i>
+              EDIT
+            </Button>
+            <Button
+              id="bed-delete"
+              className="btn-danger btn ml-auto text-white mx-2 px-2"
+              type="submit"
+              size="small"
+              onClick={() => handleDelete(name, id)}
+            >
+              <i className="fas fa-trash text-white mr-2"></i>
+              Delete
+            </Button>
+          </div>
         )}
       </td>
+      {bedData.show && (
```

Comment:
Instead of a conditionally rendering component, pass `bedData.show`  as a prop and `handleClose` callback to close `modal`

Reply:
Done! `bedData.show` is passed as a prop. Callbacks are already passed to the modal.